Remotion/Validation/Core/Validators/LengthValidator.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace Remotion.Validation.Validators
 {
-  public class LengthValidator : ILengthValidator
+  public class LengthValidator : IMinimumLengthValidator, IMaximumLengthValidator
   {
     public int Min { get; }
```

Comment:
follow-up change: Max should be greater than Min, not just greater-or-equal.

Reply:
https://re-motion.atlassian.net/browse/RM-8858